frontend/app/components/home/home-template.html

```diff
@@ -0,0 +1 @@
+<h1> homepage </h1>
```

Comment:
Not a big deal right now, but these spaces will be rendered so it depends on whether or not your design wants to have these extra spaces.